tests/devapps/XForms/XForms.Android/XForms.Android.csproj

```diff
@@ -127,7 +127,4 @@
   <Import Project="$(MSBuildExtensionsPath)\Xamarin\Android\Xamarin.Android.CSharp.targets" />
   <!-- Import the Xamarin targets workaround -->
   <Import Project="..\Patch.Xamarin.Common.targets" />
-  <PropertyGroup>
```

Comment:
why are you removing this?

Reply:
oops...deleted too much. thanks for catching this. I will update in next commit. 

---
In reply to: [246214307](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/762#discussion_r246214307) [](ancestors = 246214307)

Reply:
done.

---
In reply to: [246215224](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/762#discussion_r246215224) [](ancestors = 246215224,246214307)